Makefile

```diff
@@ -31,3 +34,17 @@ test:
 # this prevents errors like "No rule to make target 'TestFaultInjection'" when you run "make test TestFaultInjection"
 Test%:
 	@:
+
+# this will install production operators
+test.integration.install:
+	scripts/ci/operator.sh create wait get_csv
+
+# this will delete production operators
+test.integration.cleanup:
+	scripts/ci/operator.sh delete delete_csv delete_cni
+
+# In an CI job, this will run tests on a remote OpenShift cluster
+test.integration:
+	$(MAKE) test.integration.install
+	$(MAKE) test
+	$(MAKE) test.integration.cleanup
```

Comment:
I'm not sure about the term `integration` here. These are end-to-end tests, right? But the purpose of these targets is to run them as a pre-submit job in the maistra-test-tool repo, right? If so, we could rename these targets to `presubmit.install`, `presubmit.cleanup` and `presubmit`.